sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -429,21 +449,26 @@ public virtual async Task<Response<CategorizedEntityCollection>> RecognizeEntiti
             try
             {
                 var documents = new List<MultiLanguageInput>() { ConvertToMultiLanguageInput(document, language) };
-
-                Response<EntitiesResult> result = await _serviceRestClient.EntitiesRecognitionGeneralAsync(
-                    new MultiLanguageBatchInput(documents),
-                    stringIndexType: Constants.DefaultStringIndexType,
```

Comment:
this parameter is no longer passed. Could you please add it again?

Reply:
I see this one is missing in other places too. 
`StringIndexType` is what tells the service what type of encoding they should use when managing strings. (think for example when someone sends an emoji. According to the encoding, the properties of a string will change).
By default, .NET manages `Utf16codeUnit` so we always need to send this property to the service. See how the library has a default value [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/Constants.cs#L10).